docs/conf.py

```diff
@@ -0,0 +1,69 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2022.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+# pylint: disable=invalid-name
+
+"""
+Sphinx documentation builder
+"""
+
+# General options:
+import os
+
+project = "Template project"
+copyright = "2022"  # pylint: disable=redefined-builtin
+author = ""
+
+# The short X.Y version
+version = ""
```

Comment:
It would be nice to set this to _something_.  I think the most logical thing is to move these two lines below the definition of `release`, and then we can just make it `version = release`.  Searching online, it seems [quite common](https://stackoverflow.com/a/26143902/1558890) that code snippets set them to the same thing.  In fact, the [sphinx documentation](https://www.sphinx-doc.org/en/master/usage/configuration.html#project-information) says

> If you don’t need the separation provided between version and release, just set them both to the same value.